src/components/label/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, h, Prop } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-label",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyLabel {
+	@Prop() color: string
+	@Prop() name: string
+	@Prop() description: string
+	@Prop({ reflect: true }) shape: "rectangle" | "rounded"
+
+	render() {
+		return (
+			<div
+				title={this.description}
+				style={{
+					backgroundColor: `hsl(${this.color} 85% 70%)`,
+				}}>
```

Comment:
Maybe this should use the `Color` type in smoothly, e.i. `"success"`, `"danger", `"default"` and then it can be colored accordingly, instead of setting a style.
This will also not add such high specificity for this property - setting styles directly on the element can't be overwritten by css.

My suggestion:
```tsx
@Prop({reflect: true}) color: Color = "default"
```
If the global stiles work then you should not even have to add anything in `style.css` to get the color to change

Reply:
If you still want to use color like this then I suggest calling it `hue`
e.i.
```ts
@Prop() hue: number
```

Reply:
> If you still want to use color like this then I suggest calling it `hue` e.i.
> 
> ```tsx
> @Prop() hue: number
> ```

+1

Reply:
The labels do not use the theme colors (except for the text, etc)

Reply:
hue \+